infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/MachineResolver.java

```diff
@@ -68,12 +77,40 @@ public MachineResolver(
   public InternalMachineConfig resolve() throws InfrastructureException {
     InternalMachineConfig machineConfig =
         new InternalMachineConfig(
-            null, toServers(containerEndpoints), null, null, toWorkspaceVolumes(cheContainer));
+            null,
+            toServers(containerEndpoints),
+            null,
+            toMachineAttributes(pluginId, wsAttributes),
+            toWorkspaceVolumes(cheContainer));
 
     normalizeMemory(container, machineConfig);
     return machineConfig;
   }
 
```

Comment:
Adding alias attribute for cheEditor/Plugin by checking for `pluginID=alias` pair in appropriate WS config attributes;